community/values/src/main/java/org/neo4j/values/storable/DateTimeValue.java

```diff
@@ -62,6 +62,9 @@
 
 public final class DateTimeValue extends TemporalValue<ZonedDateTime,DateTimeValue>
 {
+    public static final DateTimeValue MIN_VALUE = new DateTimeValue( ZonedDateTime.of( LocalDateTime.MIN, ZoneOffset.MIN ) );
+    public static final DateTimeValue MAX_VALUE = new DateTimeValue( ZonedDateTime.of( LocalDateTime.MAX, ZoneOffset.MAX ) );
```

Comment:
Have you double checked that this doesn't overflow, because I think it is saved in UTC? Or was that only the TimeValue before you changed that in another commit of this PR?